src/stan_string.jl

```diff
@@ -0,0 +1,111 @@
+using Distributions
+function stan_string(p::Bernoulli,i)
+	parameters = (params(p)[1])
+	return string("theta[$i] ~ bernoulli($parameters)",";")
+end
+function stan_string(p::Binomial,i)
+	parameters = (params(p)[1])
+	return string("theta[$i] ~ binomial($parameters)",";")
+end
+function stan_string(p::BetaBinomial,i)
+	parameters = (params(p)[1],params(p)[2],params(p)[3])
+	return string("theta[$i] ~ beta_binomial$parameters",";")
+end
+function stan_string(p::Hypergeometric,i)
+	parameters = (params(p)[1])
+	return string("theta[$i] ~ hypergeometric($parameters)",";")
+end
+function stan_string(p::Categorical,i)
+	parameters = (params(p)[1])
+	return string("theta[$i] ~ categorical($parameters)",";")
+end
+function stan_string(p::NegativeBinomial,i)
+	parameters = (params(p)[1],params(p)[2])
+	return string("theta[$i] ~ neg_binomial$parameters",";")
+end
+function stan_string(p::Poisson,i)
+	parameters = (params(p)[1])
+	return string("theta[$i] ~ beta_binomial($parameters)",";")
+end
+function stan_string(p::Normal,i)
+	parameters = (params(p)[1],params(p)[2])
+	string("theta[$i] ~ normal$parameters",";")
+end
+function stan_string(p::TDist,i) 
+	parameters = (params(p)[1])
+	return string("theta[$i] ~ normal($parameters,0,1)",";")
```

Comment:
Huh, you're sure Stan uses Normal for thus just with a DoF part?

Reply:
Actually Distributions.jl implementation has only a DoF part https://juliastats.github.io/Distributions.jl/latest/univariate.html#Distributions.TDist

Reply:
I see. But Stan calls it a Normal distribution?